src/lib/transaction/transaction_hash.mli

```diff
@@ -51,7 +51,7 @@ include Comparable.S with type t := t
 module User_command_with_valid_signature : sig
   type hash = t [@@deriving sexp, compare, hash, yojson]
 
-  type t = private (User_command.Valid.t, hash) With_hash.t
+  type t = (User_command.Valid.t, hash) With_hash.t
```

Comment:
```suggestion
  type t = private (User_command.Valid.t, hash) With_hash.t
```

I don't think this change is necessary in the most recent version. Let's revert this line.